install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -56,6 +56,10 @@ rules:
   resources:
   - endpointslices
   verbs:
+  - create
```

Comment:
I think It would make sense to add these permissions only if this feature is enabled.
Adding additional permissions to cilium-operator by default might raise some security-concerns.